tools/issue-labeler/src/IssueLabeler/IssueLabeler.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+    <UserSecretsId>401bcdfb-b94b-448f-80ac-36ea8a447a97</UserSecretsId>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.EventHubs" Version="6.3.6" />
```

Comment:
We should trim the dependencies.   We're not using Event Hubs any longer.    We should also look at centralizing versions.